raspi-files/uninstall_otcamera.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+USER_HOME=$(getent passwd "${SUDO_USER:-$USER}" | cut -d: -f6)
+CONFIG="/boot/config.txt"
+OTCAMERA=$USER_HOME/"OTCamera"
+OTCSERVICE="/lib/systemd/system/otcamera.service"
+RCLOCAL="/etc/rc.local"
+NGINXDEFAULT="/etc/nginx/sites-available/default"
+#NGINX_DIR="/etc/nginx"
+#NGINX_SERVICE_FILE="/lib/systemd/system/nginx.service"
+#INIT_D_NGINX="/etc/init.d/nginx"
+HOSTAPD="/etc/default/hostapd"
+HOSTAPD_DIR="/etc/hostapd"
+DHCPCONF="/etc/dhcpcd.conf"
+DNSMASQCONF="/etc/dnsmasq.conf"
+HWCLOCK="/lib/udev/hwclock-set"
+
+echo "$USER_HOME"
+echo "OTCamera Uninstall Script"
+echo "############################"
+echo " "
+
+echo "Deactivate OTCamera service"
+systemctl stop otcamera.service
+systemctl disable otcamera.service
+rm $OTCSERVICE
+
+echo "Disable WiFi AP"
+sed $RCLOCAL -i -e "/\/bin\/bash \/usr\/local\/bin\/wifistart/d" 
+apt remove hostapd dnsmasq dhcpcd -y
+
+sed $HWCLOCK -i -e "/#if.*systemd.*then/s/#//g"
```

Comment:
same here: Backup hwclock-set during installation and copy back on uninstall?

Reply:
Will do!